lib/ui/top_app_bar.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import '../main.dart';
+
+class TopBar extends StatefulWidget implements PreferredSizeWidget {
+  final String title;
+
+  const TopBar(
+      {Key? key, required this.title}): super(key:key);
+
+  Size get preferredSize => Size.fromHeight(50.0);
```

Comment:
This could be a `final` variable, rather than a getter.

Reply:
Is this the changes you were thinking?

```
final Size preferredSize = Size.fromHeight(50.0);

TopBar(
      {Key? key, required this.title}): super(key:key);
```

Reply:
Yeah!
You can omit the first `Size` too, since it's implied by the type of left-hand side of the assignment.

`final preferredSize = Size.fromHeight(50.0)`